src/codeu/chat/util/TextValidator.java

```diff
@@ -0,0 +1,22 @@
+package codeu.chat.util;
+
+import java.util.regex.Pattern;
+
+/**
+ * Class used to determine whether or not certain inputs are valid for different fields throughout
+ * the program
+ */
+public final class TextValidator {
+
+  private static final Pattern validUserNamePattern = Pattern.compile("[a-zA-Z0-9]+");
+
+  public static boolean isValidUserName(String username) {
+    if (username == null) {
```

Comment:
You could consolidate this conditional a bit with 
`com.google.common.base.Strings Strings.isNullOrEmpty()`